Building with vite fails

I've added this into a vite react project, importing as instructed in the README with `import * as JoyCon from 'joy-con-webhid';` and I'm getting this error...

Any ideas?

```
✘ [ERROR] Could not resolve "./connectRingCon.js"

    node_modules/joy-con-webhid/dist/joycon.js:1:66:
      1 │ ..."./parse.js";import{connectRingCon}from"./connectRingCon.js";const concatTypedArrays=(a,b)=>{const ...
        ╵                                           ~~~~~~~~~~~~~~~~~~~~~

14:26:53 [vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/joy-con-webhid/dist/joycon.js:1:66: ERROR: Could not resolve "./connectRingCon.js"
    at failureErrorWithLog (<path>/node_modules/esbuild/lib/main.js:1649:15)
    at <path>/node_modules/esbuild/lib/main.js:1058:25
    at <path>/node_modules/esbuild/lib/main.js:1525:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

--- a/package.json
+++ b/package.json
@@ -10,7 +10,8 @@
     "build:joycon": "npx terser ./src/joycon.js --toplevel --comments /@license/ --ecma=8 -o ./dist/joycon.js",
     "build:parse": "npx terser ./src/parse.js --toplevel --comments /@license/ --ecma=8 -o ./dist/parse.js",
     "build:madgwick": "npx terser ./src/madgwick.js --toplevel --comments /@license/ --ecma=8 -o ./dist/madgwick.js",
-    "build": "npm run clean && npm run build:index && npm run build:joycon && npm run build:parse && npm run build:madgwick",
+    "build:connectRingCon": "npx terser ./src/connectRingCon.js --toplevel --comments /@license/ --ecma=8 -o ./dist/connectRingCon.js",
+    "build": "npm run clean && npm run build:index && npm run build:joycon && npm run build:parse && npm run build:madgwick && npm run build:connectRingCon",
     "copyDeps": "shx cp ./node_modules/ahrs/Madgwick.js ./src/madgwick.js && shx sed -i 's/module.exports =/export/' ./src/madgwick.js",
     "prepare": "npm run lint && npm run fix && npm run build && npm run copyDeps",
     "fix": "npx prettier --write ."

